src/prefect/client/orchestration.py

```diff
@@ -1691,53 +1682,19 @@ async def create_deployment(
 
         return UUID(deployment_id)
 
-    async def update_schedule(self, deployment_id: UUID, active: bool = True):
-        path = "set_schedule_active" if active else "set_schedule_inactive"
-        await self._client.post(
-            f"/deployments/{deployment_id}/{path}",
-        )
-
     async def set_deployment_paused_state(self, deployment_id: UUID, paused: bool):
         await self._client.patch(
             f"/deployments/{deployment_id}", json={"paused": paused}
         )
 
     async def update_deployment(
         self,
-        deployment: Deployment,
-        schedule: SCHEDULE_TYPES = None,
-        is_schedule_active: Optional[bool] = None,
+        deployment_id: UUID,
```

Comment:
This is more consistent with our other update methods so you can actually do a partial update